src/collective/liveblog/browser/templates/update.pt

```diff
@@ -140,6 +140,8 @@
             parser.pathname = parser.pathname + '/recent-updates';
             $.ajax({
                 url: parser.href,
+                // Don't show plone default spinner
+                global: false,
```

Comment:
http://api.jquery.com/jquery.ajax/
```
global (default: true)
Type: Boolean
Whether to trigger global Ajax event handlers for this request. The default is true. Set to false to prevent the global handlers like ajaxStart or ajaxStop from being triggered. This can be used to control various Ajax Events.
```